The Nightly Dev Build job failed on Friday (2025-01-03)

The Nightly Dev Build job failed on Friday (2025-01-03) in run [12606450238](https://github.com/TileDB-Inc/TileDB-R/actions/runs/12606450238)

The nightly builds are also failing on my fork.

The error occurs installation:

```
installing TileDB via building locally...
Downloading https://github.com/TileDB-Inc/TileDB/archive/refs/heads/main.zip: https://github.com/TileDB-Inc/TileDB/archive/refs/heads/main.zip  ...
Converting zip to tar.gz ...
tools/buildTileDB.sh: line 49: ../tiledb-src/bootstrap: No such file or directory
using inst/tiledb/{lib,include}
configure: creating ./config.status
config.status: creating src/Makevars
** libs
using C compiler: ‘gcc (Ubuntu 13.2.0-23ubuntu4) 13.2.0’
using C++ compiler: ‘g++ (Ubuntu 13.2.0-23ubuntu4) 13.2.0’
using C++17
g++ -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I../inst/tiledb/include   -DTILEDB_SILENT_BUILD  -I'/usr/lib/R/site-library/Rcpp/include' -I'/usr/lib/R/site-library/RcppInt64/include' -I'/usr/lib/R/site-library/nanoarrow/include'     -fpic  -g -O2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -ffile-prefix-map=/build/r-base-4sITk6/r-base-4.4.2=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection -fdebug-prefix-map=/build/r-base-4sITk6/r-base-4.4.2=/usr/src/r-base-4.4.2-1.2[40](https://github.com/TileDB-Inc/TileDB-R/actions/runs/12606450238/job/35136590615#step:8:41)4.0 -Wdate-time -D_FORTIFY_SOURCE=3   -c RcppExports.cpp -o RcppExports.o
In file included from RcppExports.cpp:4:
../inst/include/tiledb.h:8:10: fatal error: tiledb/tiledb: No such file or directory
    8 | #include <tiledb/tiledb>
      |          ^~~~~~~~~~~~~~~
compilation terminated.
make: *** [/usr/lib/R/etc/Makeconf:204: RcppExports.o] Error 1
ERROR: compilation failed for package ‘tiledb’
* removing ‘/usr/local/lib/R/site-library/tiledb’
```

The last passing build occurred the evening of [Jan 2nd](https://github.com/TileDB-Inc/TileDB-R/actions/runs/12591500433) from commit https://github.com/TileDB-Inc/TileDB-R/commit/ba40daf331e4da6c8ac7d229f3d2b0263acb77f3. The first failing build occurred the evening of [Jan 3rd](https://github.com/TileDB-Inc/TileDB-R/actions/runs/12606450238) from commit https://github.com/TileDB-Inc/TileDB-R/commit/1ceaba1c48aaae6d7a0e01f06467bb2e64239a0e.

So this means the error is likely related to the update of the default branch of the TileDB repo from `dev` to `main` (https://github.com/TileDB-Inc/TileDB-R/pull/789, https://github.com/TileDB-Inc/TileDB/pull/5407).

I suspect the following lines are the culprit. After unzipping the zipfile, the directory will be `TileDB-main/` not `TileDB-dev/`:

https://github.com/TileDB-Inc/TileDB-R/blob/bec190352ca9db6fdaf12172f2b3bee37c23379c/tools/fetchTileDBSrc.R#L42-L49

--- a/tools/ci/valgrind/buildTileDB.sh
+++ b/tools/ci/valgrind/buildTileDB.sh
@@ -20,7 +20,7 @@ echo "::group::Setup sources"
 if [ ${isrelease} -eq 0 ]; then
     git clone --single-branch --branch ${version} https://github.com/TileDB-Inc/TileDB.git TileDB-${version}
     git log --graph --pretty=format:'%h - %d %s (%cr) <%an>' --abbrev-commit | head
-elif [ ${version} = "dev" ]; then
+elif [ ${version} = "main" ]; then
     wget https://github.com/TileDB-Inc/TileDB/archive/refs/heads/${version}.zip
     unzip ${version}.zip
     rm ${version}.zip

--- a/tools/fetchTileDBSrc.R
+++ b/tools/fetchTileDBSrc.R
@@ -43,9 +43,9 @@ if (!file.exists("tiledb.tar.gz")) {
         cat("Converting zip to tar.gz ...\n")
         unzip("tiledb.tar.gz")
         unlink("tiledb.tar.gz")
-        Sys.chmod("TileDB-dev/bootstrap", mode="0755")
+        Sys.chmod("TileDB-main/bootstrap", mode="0755")
         options(warn=-1)
-        tar("tiledb.tar.gz", "TileDB-dev", compression="gzip")
+        tar("tiledb.tar.gz", "TileDB-main", compression="gzip")
     }
 
     options(op)

--- a/vignettes/introduction.Rmd
+++ b/vignettes/introduction.Rmd
@@ -182,7 +182,7 @@ column *reduces to a vector* as this happens at the R side:
 
 Basic reading returns the coordinates and any attributes. The following
 examples use the array created by the
-[quickstart_sparse](https://github.com/TileDB-Inc/TileDB/blob/dev/examples/cpp_api/quickstart_sparse.cc)
+[quickstart_sparse](https://github.com/TileDB-Inc/TileDB/blob/main/examples/cpp_api/quickstart_sparse.cc)
 example.
 
 ``` r

